components/Buttons/examples/ButtonsTypicalUse.m

```diff
@@ -134,19 +135,33 @@ - (void)viewDidLoad {
 - (void)didTap:(id)sender {
   NSLog(@"%@ was tapped.", NSStringFromClass([sender class]));
   if (sender == self.floatingButton) {
-    [self.floatingButton
-          collapse:YES
-        completion:^{
-          dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(1 * NSEC_PER_SEC)),
-                         dispatch_get_main_queue(), ^{
-                           [self.floatingButton expand:YES completion:nil];
-                         });
-        }];
+    [self.floatingButton collapse:YES
```

Comment:
nit: the indentation here makes the line harder to read, maybe worth putting some of the params in the same line?

Reply:
Done. Restored the original indentation (it is at 101 columns anyway).